property/index.js

```diff
@@ -75,6 +77,9 @@ module.exports = yeoman.Base.extend({
         name: 'type',
         message: 'Property type:',
         type: 'list',
+        default: this.propDefinition && 
+          (Array.isArray(this.propDefinition.type) ? 
+            'array' : this.propDefinition.type),
```

Comment:
Oh, ignore this comment, I choose "number" instead of "array of numbers" in the first run, my bad. The prompts work fine AFAICT 👍